scg-system/src/main/java/io/telicent/core/SmartCacheGraph.java

```diff
@@ -119,4 +132,51 @@ private static boolean isAuthEnabled() {
     private static boolean isInitialCompactionEnabled() {
         return !Configurator.get(FMod_InitialCompaction.DISABLE_INITIAL_COMPACTION, Boolean::parseBoolean, false);
     }
+
+    private static String convertToAbsolutePathURI(String fileUri) {
+        // Remove the "file:" prefix to get the relative path
+        String relativePath = fileUri.substring(5);
+
+        // Convert the relative path to an absolute path
+        File file = new File(relativePath);
+        String absolutePath = file.getAbsolutePath();
+
+        // Return the absolute path as a file URI
+        return "file:/" + absolutePath;
+    }
+
+    private static void convertYamlConfigToRDF(String... args) {
+        YAMLConfigParser ycp = new YAMLConfigParser();
+        RDFConfigGenerator rcg = new RDFConfigGenerator();
+        String configPath;
+        String pattern = ".*\\.(yaml|yml)$";
+        Pattern regex = Pattern.compile(pattern);
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].equalsIgnoreCase("--conf") && i + 1 < args.length ) {
```

Comment:
Alternative suggestion that preserves the `for` loop as-is and merely improves the `if` condition

```suggestion
            if (StringUtils.equalsAnyIgnoreCase(args[i], "--config", "--conf") && i + 1 < args.length ) {
```

That's `StringUtils` from Apache Commons Lang which should already be a dependency via Jena